apps/redi-connect/src/pages/front/signup/SignUp.tsx

```diff
@@ -179,6 +182,25 @@ export default function SignUp() {
               {...formik}
             />
 
+            {type === 'mentor' && (
+              <>
+                <Checkbox.Form
+                  name="mentor_isPartnershipMentor"
+                  checked={formik.values.mentor_isPartnershipMentor}
+                  className="submit-spacer"
+                  {...formik}
+                >
+                  My employer is in a mentorship partnership with ReDI School
+                </Checkbox.Form>
+
+                <FormInput
```

Comment:
Should only be displayed if checkbox is checked.

Reply:
This part I'll leave it to Kate as she will take on the Frontend of the task